build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -105,6 +105,11 @@ service Execution {
   // where, for each requested blob not present in the CAS, there is a
   // `Violation` with a `type` of `MISSING` and a `subject` of
   // `"blobs/{hash}/{size}"` indicating the digest of the missing blob.
+  //
+  // The server does not need to guarantee that a call to this method leads to
+  // at most one execution of the action. The server MAY execute the action
+  // multiple times, potentially in parallel. These redundant executions MAY
```

Comment:
This is probably reasonable given that we've been lax about it to this point. I can also see how parallel execution could be useful. There are other situations where it's problematic, though--think end-to-end tests that need to have a lock on external resources. At some point in the future we may need an API construct to disallow parallel execution.